msdsn/conn_str.go

```diff
@@ -232,7 +231,7 @@ func Parse(dsn string) (Config, map[string]string, error) {
 			var err error
 			p.Krb5Conf, err = setupKerbConfig(krb5ConfFile)
 			if err != nil {
-				return p, params, fmt.Errorf("cannot read kerberos configuration file: %v", err)
+				return p, params, fmt.Errorf("cannot read kerberos configuration file: %v", err.Error())
```

Comment:
This should be:
`return p, params, fmt.Errorf("cannot read kerberos configuration file: %w", err)`